assets/models/system/Mistral-7B-Instruct-v0-1/spec.yaml

```diff
@@ -0,0 +1,40 @@
+$schema: https://azuremlschemas.azureedge.net/latest/model.schema.json
+name: Mistral-7B-Instruct-v0-1
```

Comment:
Name needs to be "Mistral-7B-Instruct-v01"

Reply:
updated